amundsen_application/api/metadata/v0.py

```diff
@@ -54,14 +54,14 @@ def popular_tables() -> Response:
     """
     try:
         url = app.config['METADATASERVICE_BASE'] + POPULAR_TABLES_ENDPOINT
-        response = request_metadata(url=url)
+        params = {'limit': app.config['POPULAR_TABLE_COUNT']}
+        response = request_metadata(url=url, params=params)
```

Comment:
@ttannis FYI, this is the endpoint from metadata side: https://github.com/lyft/amundsenmetadatalibrary/blob/master/metadata_service/api/popular_tables.py#L21